cabal.project

```diff
@@ -0,0 +1,14 @@
+packages: streamly-process.cabal
+
+source-repository-package
+   type: git
+   location: https://github.com/composewell/streamly.git
+   tag: master
+
+source-repository-package
+  type: git
+  location: https://github.com/composewell/streamly.git
+  tag: master
+  subdir: core
+
+jobs: 1
```

Comment:
used for HLS to work .

Reply:
If we use correct version in default.nix, hls should work without it.

Reply:
Why we do not use cabal.project is precisely because of hls and cabal messing up with git. With this the git repo gets into detached head mode automatically.